src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -1,35 +1,32 @@
-com.minecolonies.error.supplyChestAlreadyPlaced=You have already placed a supply ship!
-
+com.minecolonies.error.supplyChestAlreadyPlaced=You have already placed a supply ship\!
```

Comment:
We do not need them escaped. You can add a line to it.

2016-08-28 23:45 GMT-03:00 Colton Schlosser <notifications@github.com>:

> In src/main/resources/assets/minecolonies/lang/en_US.lang
> <https://github.com/Minecolonies/minecolonies/pull/68#discussion_r76547341>
> :
>
> > @@ -1,35 +1,32 @@
> > -com.minecolonies.error.supplyChestAlreadyPlaced=You have already placed a supply ship!
> > -
> > +com.minecolonies.error.supplyChestAlreadyPlaced=You have already placed a supply ship\!
>
> Do we need these characters escaped? Or should I add a line to the
> fix-lang tool to remove all \ characters?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Minecolonies/minecolonies/pull/68/files/43310a009a82b3dd51467756abd297b8c7ade842#r76547341>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGI9y9hxVloilPp1zyoguv1psVtbgtIJks5qkkfegaJpZM4Ju8K->
> .
>